auth-web/src/components/auth/create-account/AccountInfoNonLinked.vue

```diff
@@ -0,0 +1,337 @@
+<template>
```

Comment:
The button action row in AccountCreateBasic may not match with the design exactly, but I've changed it to use that component.  Recommend that a ticket is created to extract button actions in to their own component instead of each component including their own set.  It is fairly hard to manage as is.